package.json

```diff
@@ -2,9 +2,11 @@
   "name": "commisery",
   "version": "1.0.0",
   "description": "Commisery Action",
-  "main": "dist/index.js",
+  "main": "dist/main/index.js",
   "scripts": {
-    "build": "ncc build -o dist src/main.ts && cp -f requirements.txt dist/requirements.txt",
+    "build": "run-p build:*",
+    "build:main": "ncc build -o dist/main src/main.ts && cp -f requirements.txt dist/main/requirements.txt",
```

Comment:
Please update the `action.yml` in the root of the repository to point to the new location of your distribution